src/gui/src/layoutViewer.h

```diff
@@ -285,9 +314,15 @@ class LayoutViewer : public QWidget, public odb::dbBlockCallBackObj
   QMenu* layout_context_menu_;
   QMap<CONTEXT_MENU_ACTIONS, QAction*> menu_actions_;
 
+  // shift required to center the block in the window
   QPoint centering_shift_;
+
+  // center of the layout in window in dbu
+  // this is needed to keep track of the center without a
+  // dependence on the resolution
   odb::Point center_;
 
+  // cache of the maximum cut size
```

Comment:
Cache of the maximum cut size per layer.  Used to determine when cuts are too small to be seen and should not be drawn.